_data/changelogs/tokens-spacing.yml

```diff
@@ -2,6 +2,19 @@ title: Spacing tokens
 type: token
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Fixed a bug that prevented `units(0)` from being used in a `calc()` function.
+    summaryAdditional: This change updated the unit-less `0` token value to have a unit when needed.
```

Comment:
Could we improve clarity here? It's not clear who's affected and how this matters.

Does this mean I can use `calc(units(0))` in my project and it will work?

Reply:
I pushed up an update for additional clarity. Let me know what you think!

```diff
-    summary: Fixed a bug that prevented `units(0)` from being used in a `calc()` function.
-   summaryAdditional: This change updated the unit-less `0` token value to have a unit when needed.
+   summary: Fixed a bug that caused an error when using `units(0)` within a `calc()` function.
+   summaryAdditional: Now, `units(0)` will return `0rem` to allow it's use in sass functions. If the unit is not needed, it is automatically dropped during compilation.
```